### PR TITLE
feat: add cloud-init deployment

### PR DESCRIPTION
**Pull request by matinzd**
Add cloud-init deployment

### CODE REVIEW

File: cloud/cloud-init.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ - git
+ 
+runcmd:
+  - sudo mkdir -p /etc/apt/keyrings
+  - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+  - echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+  - sudo apt-get update
+  - sudo apt-get install -y docker-ce docker-ce-cli containerd.io docker-compose-plugin
+  - git clone https://github.com/WhatsApp/proxy.git $HOME/whatsapp-proxy
+  - docker compose -f $HOME/whatsapp-proxy/proxy/ops/docker-compose.yml up -d
```
Comment (gmale) [inline cloud/cloud-init.yml:21]: nit: newline missing

Comment (matinzd) [reply cloud/cloud-init.yml]: Fixed.

**Comment by matinzd** [comment]
@gmale @slawlor

**Review by slawlor — CHANGES_REQUESTED** [review]
Please separate the cloud init bits to another readme in a `doc/` folder. Then just list a pointer to it in the readme. We're going to add more common deployments in the future, and I don't want the main page cluttered for the bulk of our users.

Thanks

**Comment by matinzd** [comment]
> Please separate the cloud init bits to another readme in a `doc/` folder. Then just list a pointer to it in the readme. We're going to add more common deployments in the future, and I don't want the main page cluttered for the bulk of our users.
> 
> Thanks

Done!

**Review by slawlor — APPROVED** [review]
Thank you for contributing!

File: cloud/cloud-init.yml (hunk 1 above)
Comment (Sultan12490) [reply cloud/cloud-init.yml]: Hl

**Pull request merged**